connections/package.json

```diff
@@ -8,7 +8,7 @@
     "build:testing": "ng build --configuration testing --aot --build-optimizer",
     "build": "ng build  --configuration production --aot --build-optimizer",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "jest --coverage"
```

Comment:
@Siddharth-Tiwari1712 see if you can put package-lock into the gitignore

Reply:
@michael-gokey-architect added in new PR